contrib/debian/examples/bitcoin.conf

```diff
@@ -94,6 +104,14 @@
 #rpcsslcertificatechainfile=server.cert
 #rpcsslprivatekeyfile=server.pem
 
+# Transaction Fee Changes in 0.10.0
```

Comment:
Agree with @luke-jr here. It is good to mention in the comment when a feature was added, but the organization of the file is more useful when the options are grouped by theme.